src/main/java/ReviewResponse2.java

```diff
@@ -0,0 +1,25 @@
+public class ReviewResponse2 {
+    private int line;
+    private String comment;
+
+    public ReviewResponse2(int line, String comment) {
+        this.line = line;
+        this.comment = comment;
```

Comment:
:information_source: The class name does not match the file name (ReviewResponse2). Inconsistent naming can lead to confusion and maintenance issues. (Severity: Minor)